ice40/CMakeLists.txt

```diff
@@ -17,6 +17,25 @@ function(icestorm_setup)
     )
   get_target_property(LIBFTDI_TARGET env LIBFTDI_TARGET)
 
+  add_conda_pip(
+    NAME numpy
+    NO_EXE
+    )
+
+  add_thirdparty_package(
+    NAME fasm
+    BUILD_INSTALL_COMMAND "cd ${symbiflow-arch-defs_SOURCE_DIR}/third_party/fasm && ${PYTHON3} setup.py install"
```

Comment:
`python setup.py develop` might be a better option here.

See this -> https://stackoverflow.com/questions/19048732/python-setup-py-develop-vs-install
> python setup.py install is used to install (typically third party) packages that you're not going to develop/modify/debug yourself.